SPECS/libvirt/libvirt.spec

```diff
@@ -1,654 +1,1648 @@
+
+%define with_storage_zfs      0
+
+%define with_storage_iscsi_direct 0%{!?_without_storage_iscsi_direct:1}
+# azl: to maintain azl2.0 settings, explicitly disable iscsi
+# by setting with_storage_iscsi_direct=0
+%define with_storage_iscsi_direct 0
+
+# Other optional features
+%define with_numactl          0%{!?_without_numactl:1}
+
+# A few optional bits off by default, we enable later
+%define with_fuse             0
+%define with_sanlock          0
+%define with_numad            0
+%define with_nbdkit           0
+%define with_nbdkit_config_default 0
+%define with_firewalld_zone   0
+%define with_netcf            0
+%define with_libssh2          0
+%define with_wireshark        0
+%define with_libssh           0
+%define with_dmidecode        0
+
+# Finally set the OS / architecture specific special cases
+
+# Architecture-dependent features
+%ifnarch %{arches_xen}
+    %define with_libxl 0
+%endif
+%ifnarch %{arches_vbox}
+    %define with_vbox 0
+%endif
+%ifnarch %{arches_numactl}
+    %define with_numactl 0
+%endif
+%ifnarch %{arches_zfs}
+    %define with_storage_zfs 0
+%endif
+%ifnarch %{arches_ceph}
+    %define with_storage_rbd 0
+%endif
+
+%define with_openvz 0
+%define with_vbox 0
+%define with_vmware 0
+%define with_libxl 0
+%define with_hyperv 0
+%define with_lxc 0
+
+%define with_netcf 0
+
+# fuse is used to provide virtualized /proc for LXC
+%if %{with_lxc}
+    %define with_fuse      0%{!?_without_fuse:1}
+%endif
+# azl: to maintain azl2.0 settings, explicitly enable fuse
+# by setting with_fuse=1
+%define with_fuse 1
+
+# Enable sanlock library for lock management with QEMU
+# Sanlock is available only on arches where kvm is available for RHEL
+%define with_sanlock 0%{!?_without_sanlock:1}
+
+# Enable libssh2 transport for new enough distros
+# azl: to maintain azl2.0 settings, explicitly enable libssh2
+# by setting with_libssh2=1
+%define with_libssh2 1
+
+# Enable wireshark plugins for all distros
+# azl: to maintain azl2.0 settings, explicitly disnable wireshark
+# by setting _with_wireshark=0
+%define with_wireshark 0
+%define wireshark_plugindir %(pkg-config --variable plugindir wireshark)/epan
+
+# Enable libssh transport for all distros
+%define with_libssh 0%{!?_without_libssh:1}
+# azl: to maintain azl2.0 settings, explicitly disable libssh
+# by setting with_libssh=0
+%define with_libssh 0
+
+%if %{with_qemu} || %{with_lxc}
+# numad is used to manage the CPU and memory placement dynamically,
+# it's not available on many non-x86 architectures.
+    %ifarch %{arches_numad}
+        %define with_numad    0%{!?_without_numad:1}
+    %endif
+%endif
+
+# We want to build with nbdkit support, but should only enable nbdkit by
+# default if the OS ships a SELinux policy that allows libvirt to launch it.
+# Right now that's not the case anywhere, but things should be fine by the time
+# Fedora 40 is released.
+%if %{with_qemu}
+    # bfjelds: DO WE NEED THIS?? for Fedora, it is enabled for 0%{?fedora} || 0%{?rhel} >= 9
```

Comment:
Previously, 2.0 did not include nbd at all so disabling this would not be a regression. I would suggest we keep it off in 3.0.